src/util/system.h

```diff
@@ -14,6 +14,9 @@
 #include <config/bitcoin-config.h>
 #endif
 
+/* Define to 1 if std::system or (for Windows) ::wsystem is available */
+#define HAVE_SYSTEM HAVE_STD__SYSTEM || HAVE_WSYSTEM
```

Comment:
Should this be happening in configure alone, or here alone?

Reply:
I've kept `HAVE_STD__SYSTEM` and `HAVE_WSYSTEM` separate, because it allows for separate handling of Windows vs. non-Windows. `HAVE_SYSTEM` combines the two, for code that doesn't care.

Reply:
All for keeping the separate. I'm more calling out that this definition seems to be happening in configure.ac as well: `AC_DEFINE([HAVE_SYSTEM], [HAVE_STD__SYSTEM || HAVE_WSYSTEM], [HAVE_WSYSTEM])`, and that it's preferable to have one or the other to avoid potential divergence / confusion.

Reply:
Ah oops, I'll get rid the duplicate.